### DISCUSSION TITLE
请问使用 koyeb 部署，但数据库无论是 mysql 还是 postgres 几乎全都报错，究竟是哪里出了问题？

### DISCUSSION THREAD

**Discussion by eliguoguo** [AlistGo/alist#5944] [Q&A]
配置变量中配置过很多数据库，ElephantSQL、sqlpub、supabase、neon、aiven、planetscale 等等，但除了用 db4free 运行成功，其他全都失败，提示 " Instance is unhealthy " ， " Instance has exited with code 1 " ，究竟要怎么解决这个问题？
‘
附一小段运行日志：
> Instance allocated in nomad
> Instance is starting
> Instance is unhealthy
> [36mINFO[0m[2024-01-24 16:36:16] reading config file: data/config.json        
> [36mINFO[0m[2024-01-24 16:36:16] config file not exists, creating default config file 
> [36mINFO[0m[2024-01-24 16:36:16] load config from env with prefix:            
> [36mINFO[0m[2024-01-24 16:36:16] init logrus...                                                         
> Instance is unhealthy
> Instance is unhealthy
> Instance terminated in nomad

求好心人帮忙解答！

**Comment by Hwenyi** [comment]
> 配置变量中配置过很多数据库，ElephantSQL、sqlpub、supabase、neon、aiven、planetscale 等等，但除了用 db4free 运行成功，其他全都失败，提示 " Instance is unhealthy " ， " Instance has exited with code 1 " ，究竟要怎么解决这个问题？ ‘ 附一小段运行日志：
> 
> > Instance allocated in nomad
> > Instance is starting
> > Instance is unhealthy
> > �[36mINFO�[0m[2024-01-24 16:36:16] reading config file: data/config.json
> > �[36mINFO�[0m[2024-01-24 16:36:16] config file not exists, creating default config file
> > �[36mINFO�[0m[2024-01-24 16:36:16] load config from env with prefix:
> > �[36mINFO�[0m[2024-01-24 16:36:16] init logrus...
> > Instance is unhealthy
> > Instance is unhealthy
> > Instance terminated in nomad
> 
> 求好心人帮忙解答！

同样的问题，我就直接用db4free了，hhh

**Reply by NicoChiGu** [reply]
使用supabase 或类似postgresql数据库尝试 DB_SSL_MODE设置为prefer
